bluesky_config/static_web/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Example</title>
+    </head>
+    <body>
```

Comment:
Wouldn't it be better to say this is the frontend for the Bluesky Queue Server, see the README.md for details? It is served as expected, and this is where things start getting interesting once we have a web page to trigger a plan.

Reply:
I went with the simplest hello-world I found to check that it could serve at all...